stl/inc/algorithm

```diff
@@ -9848,7 +9850,7 @@ namespace ranges {
                 if (!_STD is_constant_evaluated()) {
                     const auto _First_ptr = _STD to_address(_First);
                     const auto _Last_ptr  = _First_ptr + (_Last - _First);
-                    const auto _Result    = _CSTD __std_minmax_element(_First_ptr, _Last_ptr);
+                    const auto _Result    = _STD __std_minmax_element(_First_ptr, _Last_ptr);

```

Comment:
The idea is that they behave just like the `__std_meow_N` functions without the `N` size; renaming them would break that symmetry. That said I wouldn't grumble *too* much, as long as the names are unique.

(If I had my way there wouldn't be any `_STD` at all and we wouldn't care about the incomplete type scenario :joy_cat:)